### PR TITLE
refactor : remove map in test

### PR DESCRIPTION
**Pull request by LCaffarelli**
remove all "maps" to "for...of" in tests

### CODE REVIEW

File: bots/birthday/birthday.test.js [hunk 1]
```diff
@@ -23,14 +22,13 @@ describe('birthdayBot', () => {
...
             const todayMinusOneDay = today.minus(86_400_000);
             if (user.name === 'Gwenn Stacy') {
                 user.birthday = todayMinusOneDay.toFormat('dd/MM/yyyy');
             }
-            return user;
-        });
-        expect(birthdayBot.greetsBirthdays()).not.toContain('Joyeux anniversaire Gwenn Stacy');
+        }
     });
+    expect(birthdayBot.greetsBirthdays()).not.toContain('Joyeux anniversaire Gwenn Stacy');
```
Comment (jpiazzal) [inline bots/birthday/birthday.test.js:33]: expect should be in test :)

**Review by jpiazzal — CHANGES_REQUESTED** [review]

**Review by CodeFather44 — APPROVED** [review]

**Review by LCaffarelli — COMMENTED** [review]
fix : expect is now in test

File: bots/birthday/birthday.test.js (hunk 1 above)
Comment (LCaffarelli) [reply bots/birthday/birthday.test.js]: fixed

**Review by jpiazzal — APPROVED** [review]
all good!

**Pull request merged**